tests/Html/BlockBuilderTest.php

```diff
@@ -4,6 +4,8 @@
 
 class BlockBuilderTest extends TestCase
 {
+    protected $Block;
```

Comment:
Ideally, if we're adding defined properties to classes, we should type-hint them. Would you mind adding type hints to the other tests that have new properties added as well?

```suggestion
    protected BlockBuilder $Block;
```

Reply:
I´ve added these type-hints.